src/librustdoc/html/static/css/rustdoc.css

```diff
@@ -735,17 +735,21 @@ a {
 .anchor {
 	display: none;
 	position: absolute;
-	left: -7px;
+	left: 0;
+	background: none !important;
 }
 .anchor.field {
 	left: -5px;
 }
 .small-section-header > .anchor {
-	left: -28px;
-	padding-right: 10px; /* avoid gap that causes hover to disappear */
+	left: -15px;
+	padding-right: 8px;
 }
-.anchor:before {
-	content: '\2002\00a7\2002';
+h2.small-section-header > .anchor {
+	padding-right: 2px;
+}
+.anchor::before {
+	content: '§';
```

Comment:
What's the difference between `.anchor:before` and `.anchor::before`? Is there a reason you changed it?

Also, you changed `'\2002\00a7\2002'` to `'§'`—are they equivalent?

Reply:
Ok, IIUC, `:before` is technically incorrect according to the spec and `::before` is the spec-correct way, but in practice they should behave the same. Still curious about the other change though, especially because Python says the bytes that were there before aren't valid UTF-8 or UTF-16.

Reply:
They're not equivalent. `\2002` is a whitepace. It made the anchor positioning thing more difficult to handle.

Reply:
Ah, makes sense 👍